src/varname.jl

```diff
@@ -418,28 +418,34 @@ _issubrange(i::ConcreteIndex, j::Colon) = true
 # we preserve the status quo, but I'm confused.
 _issubrange(i::Colon, j::ConcreteIndex) = true
 
+"""Turn any `AbstractRange` into an equivalent `UnitRange` or `StepRange`."""
+_normalizerange(r::AbstractRange) = (:)(r...)
```

Comment:
Is this normalization guaranteed to work and documented somewhere?

Reply:
Ha. It doesn't even work, my test cases were flawed :D